Source/SlideOutable.swift

```diff
@@ -128,7 +128,7 @@ open class SlideOutable: ClearContainerView {
      
      The default value is `0.4`.
      */
-    @IBInspectable open var anchorFraction: CGFloat? = 0.4 {
+    @IBInspectable open var anchorFraction: CGFloat = 0.4 {
```

Comment:
This doesn't seem to be Swift 5 related change 😄 

As far as I remember `anchorFraction` is optional for a reason. It allows you to have no anchor point at all (only collapsed and expanded state).

Reply:
Ah, good to know. Swift 5 doesn't let you have optionals as `@IBInspectable`, because that type can't be represented in Objective-C apparently. I'll revert the changes to optionality then and, sadly, we will have to remove the `@IBInspectable` for this property.